packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -549,8 +554,16 @@ export class ModelScene extends Scene {
         action.crossFadeFrom(lastAnimationAction, crossfadeTime, false);
       }
 
+      action.setLoop(loopMode, repetitionCount);
+
       action.enabled = true;
-      action.play();
+
+      if (repetitionCount === Infinity) {
+        action.play();
+      } else {
+        action.reset();
```

Comment:
Is this necessary? I notice it makes for an  inconsistent behavior: normally, when  you `pause()`  and then `play()`, the animation picks up  where it left  off. However, if you instead do `play({repetitions: 1})` the animation is suddenly restarted from the beginning.

Reply:
This is an unnecessary process. I think this is the result of trial and error when it did not work properly. Sorry.